specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -237,4 +311,12 @@ interface DevBoxes {
     },
     DevBoxActionsDelayMultipleResult
   >;
+
+  @doc("Lists operations on the Dev Box which have occurred within the past 90 days.")
+  @added(APIVersions.v2024_02_01)
+  listOperations is StandardResourceOperations.ResourceList<DevBoxOperation>;
+
+  @doc("Gets an operation on a Dev Box.")
+  @added(APIVersions.v2024_02_01)
+  getOperation is StandardResourceOperations.ResourceRead<DevBoxOperation>;
```

Comment:
Wondering if these operations should have been grouped in a move generic interface if the root resource is "operations" just like the getOperationStatus operation.

Reply:
I gave a though about this, but I don't think we can unify because getOperation have different path for dev box and environment. I though about having a shared model "SharedOperation" with DevBoxOperation and EnvironmentOperation extending it, and then have `getOperation is StandardResourceOperations.ResourceRead<SharedOperation>`, but "SharedOperation" would need to have different parentResouce (DevBox and Environments) for each case, so we can't actually share the operation here

Reply:
path for DevBoxOperations is `/projects/users/devboxes/operations`  and for EnvironmentOperation is `/projects/users/environments/operations`

Reply:
Got it, thanks for the info!